activesupport/test/cache/stores/redis_cache_store_test.rb

```diff
@@ -507,6 +507,11 @@ class ClearTest < StoreTest
   end
 
   class RawTest < StoreTest
+    test "configuring store with 'raw: true'" do
+      cache = lookup_store(raw: true)
+      assert_equal("bar", cache.fetch("foo") { "bar" })
+    end
```

Comment:
Should we put this in the generic store behavior test?

Reply:
yes, updated.